api/src/main/java/org/openmrs/PersonAttribute.java

```diff
@@ -42,7 +42,7 @@
  * @see org.openmrs.Attributable
  */
 @Indexed
-public class PersonAttribute extends BaseChangeableOpenmrsData implements java.io.Serializable, Comparable<PersonAttribute> {
+public class PersonAttribute extends BaseAttribute<PersonAttributeType, Person> implements Serializable, Comparable<PersonAttribute> {
```

Comment:
```suggestion
public class PersonAttribute extends BaseAttribute<PersonAttributeType, Person> implements Serializable {
```

Here, we want to remove the `Comparable` interface because it is already set by `BaseAttribute`.

Reply:
@ibacher Thanks